src/dashboard/src/components/charts/storageTrends/StorageTrendsChart.tsx

```diff
@@ -56,6 +63,8 @@ export const StorageTrendsChart: React.FC<LineChartProps> = ({
   const { isReady: serverHistoryItemsIsReady, findServerHistoryItems } = useServerHistoryItems();
   const { tenantId, organizationId, operatingSystemItemId, serverItemKey } = useDashboard();
 
+  const [forceFetch, setForceFetch] = React.useState(0);
```

Comment:
Clicking the Update button doesn't do anything if the date doesn't change.  This allows it to force an update.